azext_iot/operations/events3/_events.py

```diff
@@ -76,33 +77,56 @@ def _get_conn_props():
         properties["platform"] = sys.platform
         return properties
 
-    if not target.get('events'):
-        endpoint = _build_iothub_amqp_endpoint_from_target(target)
-        _, update = await evaluate_redirect(endpoint)
-        target['events'] = update['events']
+    if not target:
+        logger.debug('No Event Hub target found provided.')
+        return
+
+    if not target.get('central'):
```

Comment:
If checking existence of key, its more efficient to do `if 'central' not in target:`